hydrography-approach/processing_scripts/filter_data/filter_osm_ways.py

```diff
@@ -50,4 +50,4 @@ def filter_ways(input_osm_pbf, output_osm_pbf, output_gpkg):
     # Convert the filtered OSM PBF file to a GeoPackage
     convert_to_geopackage(output_osm_pbf, output_gpkg)
 
-    print(f"Output file: {output_gpkg} has been created successfully!")
+    logger.info(f"Output file: {output_gpkg} has been created successfully!")
```

Comment:
Exceptions are not handled in the file. In other scripts also, exceptions were not handled.

Reply:
I've added exception handling to the scripts.